libfwupdplugin/fu-device.h

```diff
@@ -204,6 +204,8 @@ FuDevice	*fu_device_new				(void);
  * @FU_DEVICE_INTERNAL_FLAG_MD_SET_VERFMT:		Set the device version format from the metadata if available
  * @FU_DEVICE_INTERNAL_FLAG_MD_SET_ICON:		Set the device icon from the metadata if available
  * @FU_DEVICE_INTERNAL_FLAG_RETRY_OPEN:			Retry the device open up to 5 times if it fails
+ * @FU_DEVICE_INTERNAL_FLAG_REPLUG_MATCH_GUID:		Match GUIDs on device replug
+ * @FU_DEVICE_INTERNAL_FLAG_COLDPLUG_MATCH_GUID:	Match GUIDs on device coldplug
```

Comment:
For any future plugin authors I think that this needs an explicit paragraph in documentation contrasting default behavior to this and the other flag.  It's all super confusing otherwise.